src/less/listbox/ds4/listbox-base.less

```diff
@@ -1,145 +0,0 @@
-// listbox text should NOT scale with root font-size, so all font-size defined with px
```

Comment:
I don't think we had officially decided to delete this yet. Leaving it in for backwards compatibility doesn't hurt.

Reply:
Okay, I'll put it back.